src/components/FilterBar.jsx

```diff
@@ -86,4 +86,5 @@ export default function FilterBar(props) {
 FilterBar.propTypes = {
   changeSortBy: PropTypes.func.isRequired,
   changeSortFormat: PropTypes.func.isRequired,
+  amountFilterVals: PropTypes.objectOf(PropTypes.number).isRequired,
```

Comment:
Might be good to just receive the entire query params here since all their values will likely be need to initialize various filters.